include/aikido/statespace/SO2.hpp

```diff
@@ -71,78 +34,109 @@ class SO2 : virtual public StateSpace
   /// \return new \c ScopedState
   ScopedState createState() const;
 
-  /// Gets state as a rotation angle.
+  /// Returns state as a rotation angle in (-pi, pi].
   ///
-  /// \param _state input state
-  /// \return rotation angle
-  double getAngle(const State* _state) const;
+  /// \param[in] state State.
+  double toAngle(const State* state) const;
 
-  /// Sets state to a rotation angle.
+  /// Sets state from a rotation angle.
   ///
-  /// \param _state input state
-  /// \param _angle rotation angle
-  void setAngle(State* _state, double _angle) const;
+  /// \param[out] state State corresponding to angle
+  /// \param[in] angle Rotation angle.
```

Comment:
Can we specify the range for `angle` here? (If it allows (-inf, inf), mention that)